libs/form-builder/src/components/Logger.tsx

```diff
@@ -0,0 +1,13 @@
+import { useEffect } from 'react';
+import { useWatch } from 'react-hook-form';
+
+export const Logger = ({ log }: { log: boolean }) => {
+  const formValues = useWatch();
+  useEffect(() => {
+    if (log && formValues) {
```

Comment:
I see we're handling if log is false here, but wonder if we'd need to handle it in the FormBuilder where log is called. Probably not a big deal as it would be handled at this component but wondering if it would save us an unnecessary render of Logger if log={false}

Reply:
Probably cleaner. I can clean this up more.